src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,16 @@
+dataverse.db.host=localhost
+dataverse.db.port=5432
+dataverse.db.user=dataverse
+dataverse.db.name=dataverse
+#dataverse.db.pool-resize-quantity=2
+#dataverse.db.is-connection-validation-required=false
+#dataverse.db.connection-validation-method=table
+#dataverse.db.validation-table-name=setting
+#dataverse.db.validation-classname=org.glassfish.api.jdbc.validation.PostgresConnectionValidation
+#dataverse.db.connection-leak-timeout-in-seconds=0
+#dataverse.db.connection-leak-reclaim=false
+#dataverse.db.statement-timeout-in-seconds=-1
+#dataverse.db.statement-leak-timeout-in-seconds=0
+#dataverse.db.statement-leak-reclaim=false
+#dataverse.db.slow-query-threshold-in-seconds=-1
+#dataverse.db.log-jdbc-calls=false
```

Comment:
My understanding is that all these commented out entries are for after a pull request for https://github.com/payara/Payara/issues/5024 is created and merged. They're a little confusing to see now if they don't work yet so I'd suggest removing them. Otherwise, Dataverse admins might waste time trying to get them to work.

Reply:
Removed in c583909.